### PR TITLE
Automatically choose x,y for plots

### PR DESCRIPTION
**Pull request by dcherian**
See #15

### CODE REVIEW

File: doc/whats-new.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ What's New
 v0.4.1 (unreleased)
 ===================
 
+- Automatically set ``x`` or ``y`` for ``DataArray.cf.plot``. By `Deepak Cherian`_.
```
Comment (dcherian) [inline doc/whats-new.rst:9]:
```suggestion
- Automatically set ``x`` or ``y`` for :py:attr:`DataArray.cf.plot`. By `Deepak Cherian`_.
```

File: cf_xarray/tests/test_accessor.py [hunk 2]
```diff
@@ -716,3 +744,34 @@ def test_drop_dims(ds):
...
+    assert _possible_x_y_plot(airds.air.isel(time=1), "x") == "lon"
+    assert _possible_x_y_plot(airds.air.isel(time=1), "y") == "lat"
+    assert _possible_x_y_plot(airds.air.isel(lon=1), "y") == "lat"
+    assert _possible_x_y_plot(airds.air.isel(lon=1), "x") == "time"
+
+    # choose coordinates over axes
+    assert _possible_x_y_plot(popds.UVEL, "x") == "ULONG"
+    assert _possible_x_y_plot(popds.UVEL, "y") == "ULAT"
+    assert _possible_x_y_plot(popds.TEMP, "x") == "TLONG"
+    assert _possible_x_y_plot(popds.TEMP, "y") == "TLAT"
```
Comment (malmans2) [inline cf_xarray/tests/test_accessor.py:762]: Isn't also the airds case choosing coordinates over axes (although in that case they're the same)?

Comment (dcherian) [reply cf_xarray/tests/test_accessor.py:762]: yeah this is testing for the `"coordinates"` attribute

File: doc/whats-new.rst [hunk 3]
```diff
@@ -6,7 +6,8 @@ What's New
 v0.4.1 (unreleased)
 ===================
 
-- Added scripts to document :ref:`criteria` with tables. By `Mattia Almansi`_.
+- Automatically set ``x`` or ``y`` for :py:attr:`DataArray.cf.plot`. By `Deepak Cherian`_.
+- Added scripts to document CF criteria with tables. By `Mattia Almansi`_.
```
Comment (dcherian) [inline doc/whats-new.rst:10]:
```suggestion
- Added scripts to document :ref:`criteria` with tables. By `Mattia Almansi`_.
```

**Pull request merged**